### ISSUE TITLE
Can I use the transformers.AutoTokenizer to load the tokenizer?

### ISSUE THREAD

**Issue by tian969** [meta-llama/llama3#151]
I know the tokenizer.py in this Repo use TikTokenizer, can I use transformers.AutoTokenizer to load the tokenizer so that I dont need to amend my code class? And if i not use tokenizer.py, ChatFormat can not be used too.

**Comment by tian969** [comment]
I mean transformers.PretrainedTokenizer class

**Comment by ppaanngggg** [comment]
same question

**Comment by ppaanngggg** [comment]
I find the solution, you should use model files on huggingface. There is a tokenizer.json file can be loaded directly.

**Comment by subramen** [comment]
Yes, you can use AutoTokenizer.from_pretrained('meta-llama/Meta-Llama-3-8B-Instruct)
